test_scripts/API/Navigation/SendLocation/020_SendLocation_boundary_by_parameters.lua

```diff
@@ -0,0 +1,280 @@
+    end)
+    :ValidIf (function(_,data)
+        local ErrorMessage = ""
+        local ValidationStatus = true
+        for i,_ in pairs(DateTimeParams) do
+            if data.params.timeStamp[DateTimeParams[i].name] and
+             not params.timeStamp[DateTimeParams[i].name] then
+                ErrorMessage = ErrorMessage .. DateTimeParams[i].name .. "\n"
+                ValidationStatus = false
+            end
+        end
+
+        if ValidationStatus == false then
+            return false, "SDL sent redundant address parameters to HMI:\n" .. ErrorMessage
+        else
+            return true
+        end
+    end)
+
+    self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonSendLocation.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonSendLocation.start)
+runner.Step("RAI, PTU", commonSendLocation.registerApplicationWithPTU)
+runner.Step("Activate App", commonSendLocation.activateApp)
+runner.Step("Upload file", commonSendLocation.putFile, {"a"})
+runner.Step("Upload file", commonSendLocation.putFile, {string255Char})
+
+
+--[[ Test ]]
+--[[ Lower bound set]]
+runner.Title("Test")
+runner.Title("Lower bound value")
+for key,value in pairs(boundValues) do
+    runner.Step("SendLocation_lower_" .. tostring(key), sendLocation,
+        { key, value.lower })
+end
+
+runner.Step("SendLocation_lower_locationImage", sendLocationImage,
+        { locationImage.lower })
+
+for _, value in pairs (addressParams) do
+    runner.Step("SendLocation_lower_address_" .. tostring(value), sendLocationAddress,
+        { value, "a" })
+end
+
+for key, value in pairs(DateTimeParams) do
+    runner.Step("SendLocation_lower_timeStamp_" .. tostring(key), sendLocationTimeStamp,
+        { key, value.lower})
+end
+
+--[[ Upper bound set]]
+runner.Title("Upper bound value")
+for key,value in pairs(boundValues) do
+    runner.Step("SendLocation_upper_" .. tostring(key), sendLocation,
+        { key, value.upper })
+end
+
+runner.Step("SendLocation_upper_locationImage", sendLocationImage,
+        { locationImage.upper })
+
+for _, value in pairs (addressParams) do
+    runner.Step("SendLocation_upper_address_" .. tostring(value), sendLocationAddress,
+        { value, string200Char })
+end
+
+for key, value in pairs(DateTimeParams) do
+    runner.Step("SendLocation_upper_timeStamp_" .. tostring(key), sendLocationTimeStamp,
+        { key, value.upper})
+end
+
+runner.Step("SendLocation_address_array_max_size", sendLocation,{ "addressLines", addressLinesArrayUpperBoundSize})
+runner.Step("SendLocation_address_array_max_size_max_value", sendLocation,
+    { "addressLines", addressLinesArrayUpperBoundSizeValue})
+
+for _, value in pairs(deliveryModeEnum) do
+    runner.Step("SendLocation_deliveryMode_enum_" .. tostring(value), sendLocation,
+        { "deliveryMode",_, value})
```

Comment:
@GetmanetsIrina Please get rid of '_' as values for function's parameters. Use nil instead. 
In this particular case according to definition of sendLocation(parameter, value, self) function underscore have to be removed. Otherwise script fails to execute with aborted status

Reply:
Done